File: app/src/main/java/sabalan/paydar/mohtasham/ezibazi/model/Photo.java
package sabalan.paydar.mohtasham.ezibazi.model;

import org.json.JSONException;
import org.json.JSONObject;

public class Photo {

  private int id;
  private String url;
  private String type;
  private int width;
  private int height;

  public static class Parser{
    public static Photo parse(JSONObject jsonObject){
      Photo photo = new Photo();
      try {
        photo.setId(jsonObject.getInt("id"));
        photo.setUrl(jsonObject.getString("url"));
        photo.setType(jsonObject.getString("type"));
        photo.setWidth(jsonObject.getInt("width"));
        photo.setHeight(jsonObject.getInt("height"));

      } catch (JSONException e) {
        e.printStackTrace();
      }
      return photo;
    }
  }

  public String getType() {
    return type;
  }

  public void setType(String type) {
    this.type = type;
  }

  public int getId() {
    return id;
  }

  public void setId(int id) {
    this.id = id;
  }

  public String getUrl() {
    return url;
  }

  public void setUrl(String url) {
    this.url = url;
  }

  public int getWidth() {
    return width;
  }

  public void setWidth(int width) {
    this.width = width;
  }

  public int getHeight() {
    return height;
  }

  public void setHeight(int height) {
    this.height = height;
  }


}

File: app/src/main/java/ir/easybazi/model/Photo.kt
package ir.easybazi.model

import org.json.JSONException
import org.json.JSONObject

class Photo {

    var id: Int = 0
    var url: String? = null
    var type: String? = null
    var width: Int = 0
    var height: Int = 0

    object Parser {
        fun parse(jsonObject: JSONObject): Photo {
            val photo = Photo()
            try {
                photo.id = jsonObject.getInt("id")
                photo.url = jsonObject.getString("url")
                photo.type = jsonObject.getString("type")
                photo.width = jsonObject.getInt("width")
                photo.height = jsonObject.getInt("height")

            } catch (e: JSONException) {
                e.printStackTrace()
            }

            return photo
        }
    }


}
